packages/file-upload/src/FileRejection.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { FileRejection } from './FileRejection';
+
+export default {
```

Comment:
Hey @nathsimpson these stories are currently crashing, I think you its. because you are not passing a `fileSize.`